tests/ops/integration_tests/saas/test_rollbar_task.py

```diff
@@ -0,0 +1,164 @@
+    graph = DatasetGraph(merged_graph)
+
+    v = await graph_task.run_access_request(
+        privacy_request,
+        policy,
+        graph,
+        [rollbar_connection_config],
+        {"email": rollbar_identity_email},
+        db,
+    )
+
+    assert_rows_match(
+        v[f"{dataset_name}:projects"],
+        min_size=1,
+        keys=[
+            "id",
+            "account_id",
+            "name",
+        ],
+    )
+    assert_rows_match(
+        v[f"{dataset_name}:project_access_token"],
+        min_size=1,
+        keys=[
+            "project_id",
+            "name",
+            "access_token",
+        ],
+    )
+    assert_rows_match(
+        v[f"{dataset_name}:instances"],
+        min_size=1,
+        keys=[
+            "id",
+            "timestamp",
+            "data",
+        ],
+    )
+    # verify we only returned data for our identity email
+    for instance in v[f"{dataset_name}:instances"]:
+        assert instance["data"]["person"]["email"] == rollbar_identity_email
+
+
+@pytest.mark.integration_saas
+@pytest.mark.integration_zendesk
+@pytest.mark.asyncio
+async def test_rollbar_erasure_request_task(
+    db,
+    policy,
+    erasure_policy_string_rewrite,
+    rollbar_connection_config,
+    rollbar_dataset_config,
+    rollbar_erasure_identity_email,
+    rollbar_erasure_data,
+) -> None:
+    """Full erasure request based on the zendesk SaaS config"""
+    config.execution.masking_strict = False  # Allow Delete
+
+    privacy_request = PrivacyRequest(
+        id=f"test_rollbar_erasure_request_task_{random.randint(0, 1000)}"
+    )
+    identity_kwargs = {"email": rollbar_erasure_identity_email}
+
+    identity = Identity(**identity_kwargs)
+    privacy_request.cache_identity(identity)
+
+    dataset_name = rollbar_connection_config.get_saas_config().fides_key
+    merged_graph = rollbar_dataset_config.get_graph()
+    graph = DatasetGraph(merged_graph)
+
+    v = await graph_task.run_access_request(
+        privacy_request,
+        policy,
+        graph,
+        [rollbar_connection_config],
+        identity_kwargs,
+        db,
+    )
+
+    assert_rows_match(
+        v[f"{dataset_name}:projects"],
+        min_size=1,
+        keys=[
+            "id",
+            "account_id",
+            "name",
+        ],
+    )
+    assert_rows_match(
+        v[f"{dataset_name}:project_access_token"],
+        min_size=1,
+        keys=[
+            "project_id",
+            "name",
+            "access_token",
+        ],
+    )
+    assert_rows_match(
+        v[f"{dataset_name}:instances"],
+        min_size=1,
+        keys=[
+            "id",
+            "timestamp",
+            "data",
+        ],
+    )
+    # verify we only returned data for our identity email
+    for instance in v[f"{dataset_name}:instances"]:
+        assert instance["data"]["person"]["email"] == rollbar_erasure_identity_email
+
+    temp_masking = config.execution.masking_strict
```

Comment:
This stores the current value of `config.execution.masking_strict` but we never reset it, you need to add this after the test
```
config.execution.masking_strict = masking_strict
```